graphenv/space_util.py

```diff
@@ -158,14 +158,18 @@ def _(target: collections.abc.Mapping):
 
 
 @flatten_first_dim.register(tf.Tensor)
+@tf.function
 def _(target: tf.Tensor):
     shape = tf.shape(target)
-    dest_shape = (shape[0] * shape[1], *shape[2:])
+    dest_shape = [shape[0] * shape[1]]
+    for i in range(2, len(shape)):
+        dest_shape += [shape[i]]
+
```

Comment:
@CharlesEdisonTripp, we don't have unittests for this function (although the downstream tests seem to still pass). Does this still look OK to you? tensorflow was complaining about the '*' operation and iterating over tensors in graph mode